File: introduction/ITP1_2_B-Range.cs
using System;

public class Program {
  public static void Main() {
    var abc = Console.ReadLine().Split();

    int a = Int32.Parse(abc[0]);
    int b = Int32.Parse(abc[1]);
    int c = Int32.Parse(abc[2]);

    if (a < b && b < c) {
      Console.WriteLine("Yes");
    } else {
      Console.WriteLine("No");
    }
  }
}

File: introduction/ITP1_2_B-Range.rs
use std::io;
fn main() {
  let mut input = String::new();
  io::stdin().read_line(&mut input).unwrap();
  let abc:Vec<i32> = input.trim().split(" ").map(|s| s.parse().unwrap()).collect();
  if abc[0] < abc[1] && abc[1] < abc[2] {
    println!("Yes");
  } else {
    println!("No");
  }
}
